app/controllers/questions_controller.rb

```diff
@@ -55,6 +55,20 @@ def feedback
     end 
   end
 
+  def upvote
+    current_user.vote_for(@question)
+    respond_to do |format|
+      format.js { render json: @question.votes_for - @question.votes_against }
+    end
+  end
+
+  def downvote
+    current_user.vote_against(@question)
+    respond_to do |format|
+      format.js { render json: @question.votes_for - @question.votes_against}
```

Comment:
Simpler to use `question.plusminus`: see https://github.com/bouchard/thumbs_up#lower-level-queries